packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -480,9 +480,7 @@ export class TeamsBotImpl {
     );
 
     await handler?.next(ProgressBarConstants.DEPLOY_STEP_ZIP_FOLDER);
-    const zipBuffer = utils.zipAFolder(packDir, DeployConfigs.UN_PACK_DIRS, [
-      `${FolderNames.NODE_MODULES}/${FolderNames.KEYTAR}`,
```

Comment:
If some constants such as FolderNames.KEYTAR are no longer used, we can safely remove them.